docs/symfony-integration.md

```diff
@@ -142,11 +142,17 @@ Since we are looking to export traces to both Zipkin and Jaeger we have to make
 $jaegerExporter = new JaegerExporter(
     'Hello World Web Server Jaeger',
     'http://localhost:9412/api/v2/spans'
+    new \GuzzleHttp\Client();
+    new \GuzzleHttp\Psr7\HttpFactory();
```

Comment:
Is there a reason you're creating 2 `HttpFactory()` calls here?

Reply:
None, except that we already had multiple calls in an existing example. I've changed both the old and new ones to reuse a single object.